File: ABC/176/b.go
package main
import (
  "fmt"
  "strings"
  "strconv"
  "log"
)
func main(){
  var s string
  fmt.Scan(&s)
  str := strings.Split(s, "")
  sum := 0
  for i := 0; i < len(str); i++ {
    a, err := strconv.Atoi(str[i])
    if err != nil {
      log.Fatal(err)
    }
    sum += a
  }
  if (sum % 9 == 0) {
    fmt.Println("Yes")
  } else {
    fmt.Println("No")
  }
}

File: ABC/176/b.c
int main()
{
  char s[2000000];
  scanf("%s", s);
  int i;
  long sum = 0;
  for (i = 0; s[i] != '\0'; i++)
    sum += s[i] - '0';
  if (sum % 9 == 0)
    printf("Yes");
  else
    printf("No");
}
